plugins/polio/js/src/forms/RoundForm.tsx

```diff
@@ -70,6 +71,14 @@ export const RoundForm: FunctionComponent<Props> = ({ roundNumber }) => {
                     />
                 </Grid>
                 <Grid xs={12} md={6} item>
+                    <LqasDistrictsPassed
+                        lqasDistrictsFailing={
+                            rounds[roundIndex].lqas_district_failing
```

Comment:
i would double check the presence of `rounds[roundIndex]` , it shouldn't happen but ...